charts/posthog/values.yaml

```diff
@@ -485,8 +485,8 @@ kafka:
       - posthog-posthog-zookeeper:2181
 
 externalKafka:
-  # - External Kafka brokers. Ignored if `kafka.enabled` is set to `true`. Multiple brokers can be provided in a comma separated list, e.g. `host1:port1,host2:port2`.
-  brokers: ""
+  # - External Kafka brokers. Ignored if `kafka.enabled` is set to `true`. Multiple brokers can be provided as array/list.
```

Comment:
This is not a breaking change as we didn't cut a release (see https://github.com/PostHog/charts-clickhouse/pull/271)